src/featbin/wav-reverberate.cc

```diff
@@ -133,22 +176,37 @@ int main(int argc, char *argv[]) {
     po.Register("noise-channel", &noise_channel,
                 "Specifies the channel of the noise file, "
                 "it will only be used when multi-channel-output is false");
-    po.Register("noise-file", &noise_file,
-                "File with additive noise");
-    po.Register("snr-db", &snr_db,
-                "Desired SNR(dB) of the output");
+    po.Register("impulse-response", &rir_file,
+                "File with the impulse response for reverberating the input wave");
+    po.Register("additive-signals", &additive_signals,
+                "A comma separated list of additive signals");
```

Comment:
are these rxfilenames of wav files of additive signals?  In any case, best to give an example.